internal/commands/config_pull_policy.go

```diff
@@ -0,0 +1,81 @@
+package commands
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	pubcfg "github.com/buildpacks/pack/config"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/internal/style"
+	"github.com/buildpacks/pack/logging"
+)
+
+func ConfigPullPolicy(logger logging.Logger, cfg config.Config, cfgPath string) *cobra.Command {
```

Comment:
I went with `ConfigPullPolicy`, even though the purpose of the function is a combination of configuring and setting a default. I could change it to `SetDefaultPullPolicy`, or possibly split it into two. Let me know if there might be preferences.